### DISCUSSION TITLE
TCP raw_message_key not documented anywhere

### DISCUSSION THREAD

**Discussion by dlbarduzzi** [fluent/fluent-bit#7604] [General]
Hi,

While working with the TCP output, I could not find a way to send a particular value (a field) from my record only instead of the full record. Then I found the raw_message_key option while playing the with the TCP output (I added a body_key from the HTTP output to see what could happen). Then I go an error with the all the keys that I could use with TCP. When the raw_message_key was the perfect key I was looking/trying to do for a good time. I guess my "discussion" is simply a question about why isn't this documented? Is it deprecated (does't seem like it since it was added with 2.0.9)? Or just missed somewhere? Anyway, I was just curious and I'm very happy to have found that key.

Thanks for this great work you guys do with fluent-bit.
Dylan

**Comment by patrick-stephens** [comment]
You can get all the keys via the --help option to the plugin.

Feel free to contribute a docs update for it though, it sounds like it was just missed.

https://github.com/fluent/fluent-bit-docs
